pkg/crc/machine/bundle/metadata.go

```diff
@@ -256,12 +256,12 @@ func getBundleDownloadInfo() (*download.RemoteFile, error) {
 	return downloadInfo, nil
 }
 
-func Download() error {
-	downloadInfo, err := getBundleDownloadInfo()
+func Download(preset preset.Preset) error {
+	downloadInfo, err := getBundleDownloadInfo(preset)
 	if err != nil {
 		return err
 	}
-	if _, err := downloadInfo.Download(constants.GetDefaultBundlePath(preset.OpenShift), 0664); err != nil {
+	if _, err := downloadInfo.Download(constants.GetDefaultBundlePath(preset), 0664); err != nil {
```

Comment:
Looks like this could be merged with the previous commit? This does not seem to add a lot more new lines, it's mostly changing lines which were already changed in the previous patch.